# HG changeset patch
# User Kai T. Ohlhus <k.ohlhus@gmail.com>
# Date 1491379706 -7200
#      Wed Apr 05 10:08:26 2017 +0200
# Node ID 7dc148266dca3e1f58ac73933d2bc109be1fd7f0
# Parent  49f051ef6f2fdbb78be5d9da1b1ccdbef69fe495
Avoid initial newline in disp() for cell arrays (bug #50683).

libinterp/octave-value/ov-cell.cc (octave_cell::print_raw): Do not print
initial newline to have a consistent disp with other arrays.

diff --git a/libinterp/octave-value/ov-cell.cc b/libinterp/octave-value/ov-cell.cc
--- a/libinterp/octave-value/ov-cell.cc
+++ b/libinterp/octave-value/ov-cell.cc
@@ -638,17 +638,16 @@ octave_cell::print_raw (std::ostream& os
 
   if (nd == 2)
     {
       octave_idx_type nr = rows ();
       octave_idx_type nc = columns ();
 
       if (nr > 0 && nc > 0)
         {
-          newline (os);
           indent (os);
           os << "{";
           newline (os);
 
           increment_indent_level ();
 
           for (octave_idx_type j = 0; j < nc; j++)
             {
